src/Components/App/App.js

```diff
@@ -8,38 +7,68 @@ import Login from '../Login/Login';
 import Home from '../Home/Home';
 import ViewPacks from '../ViewPacks/ViewPacks';
 import Pack from '../Pack/Pack';
+import LikedNames from '../LikedNames/LikedNames';
 import NotFoundPage from '../NotFoundPage/NotFoundPage';
+import userData from '../../Data/User';
+import './App.css';
 
 class App extends Component {
   constructor() {
     super();
       this.state = {
-
+        allUsers: userData.users,
+        currentUser: null,
+        likedNames: [],
+        allPacks: [],
+        loggedIn: false
       }
   }
 
+  setUser = (user) => {
+    this.setState({ currentUser: user });
+    this.setState({ likedNames: user.likedNames });
+    this.setState({ allPacks: user.packs });
+    this.setState({ loggedIn: true })
+  }
+
+  logoutUser = () => {
+    this.setState({ currentUser: null });
+    this.setState({ likedNames: [] });
+    this.setState({ allPacks: [] });
+    this.setState({ loggedIn: false });
+  }
+
```

Comment:
nice method of resetting state, excited to see how we'll setState once we're connected to BE